src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const shotTerm = 2;
+  const longTerm = 7;
```

Comment:
we usually name constants in upper case
<img width="473" alt="image" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/70220667/2a3cc016-0037-452b-849f-9bb846e5ffc3">